en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
+  Can be used to configure aerodynamic surface as flap or to compensate for generated torque through main flaps.
+- (Advanced) `Spoiler Scale`: How much this actuator is deflected at the "full spoiler configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as spoiler or to compensate for generated torque through main spoiler.
+- (VTOL only) `Lock control surfaces in hover`:
   - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
   - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
 
+#### Flaps and Spoilers Configuration
+
+`Flap-control` and `spoiler-control` are aerodynamic configurations that can either be commanded manually by the pilot through a switch on the RC, or are set automatically by the controller, e.g. to reduce the airspeed before landing. 
+The flap and spoiler configuration controls are of the range of [0, 1] and can then be arbitrarily allocated to the control surfaces.
+
+It's important to understand the difference: with `flap/spoiler-control` we refer to an abstract metric of "how much flap/spoiler configuration" the controller currently demands, while the actual flaps or spoiler actuator setpoint can vary from that value.
+
+In the following example, the vehicle has two ailerons, one elevator, one rudder and two flaps as control surfaces:
+
+![Flaps and spoiler actuator configuration example](../../assets/config/actuators/qgc_actuators_tab_flaps_spoiler_setup.png)
+
+- The flaps have both `Flap Scale` set to 1, meaning that they will be fully deflected with the flap-control at 1.
+  They also have a slew rate of 0.5/s, meaning that it will take 2s to fully deflect them (a slew rate on the flaps is generally recommended to reduce the disturbances their movement creates).
+- The ailerons are primarily tasked to provide the commanded roll torque.
+  They also have `Spoiler Scale` set to 0.5, and will additionally be deflected upwards 50% if the controller demands full spoiler configuration.
+  The aileron deflection is thus the sum of the (asymmetrical) deflection for the roll torque, plus the (symmetrical) deflection for the spoiler setpoint.
+- The elevator is primarily tasked to provide pitch torque.
+  It also has non-zero entries in the `Flap Scale` and `Spoiler Scale` fields.
+  These are the elevator deflections added to compensate for the pitching moments generated by the flaps and spoiler actuators.
+  In the case here the elevator would be deflected 0.3 up when the flaps are fully deployed to counteract the pitching down moment caused by the flaps. 
 
```

Comment:
Note, all of the above explains what they are, which is good. Say I want to set the flaps scale for an aileron though, how do I decide "0.3" would be a good value? Are there rules of thumb, or tuning guides?

Reply:
There is no easy rule of thumb, though we could mention that one should be careful with not deflecting the ailerons too much to keep good effectiveness for roll control